Mathlib/Data/Complex/Basic.lean

```diff
@@ -721,62 +807,70 @@ theorem normSq_div (z w : ℂ) : normSq (z / w) = normSq z / normSq w :=
 
 
 @[simp, norm_cast]
-theorem of_real_nat_cast (n : ℕ) : ((n : ℝ) : ℂ) = n :=
+theorem ofReal_nat_cast (n : ℕ) : ((n : ℝ) : ℂ) = n :=
   map_natCast ofReal n
-#align complex.of_real_nat_cast Complex.of_real_nat_cast
+#align complex.of_real_nat_cast Complex.ofReal_nat_cast
 
 @[simp, norm_cast]
-theorem nat_cast_re (n : ℕ) : (n : ℂ).re = n := by rw [← of_real_nat_cast, of_real_re]
+theorem nat_cast_re (n : ℕ) : (n : ℂ).re = n := by rw [← ofReal_nat_cast, ofReal_re]
 #align complex.nat_cast_re Complex.nat_cast_re
 
 @[simp, norm_cast]
-theorem nat_cast_im (n : ℕ) : (n : ℂ).im = 0 := by rw [← of_real_nat_cast, of_real_im]
+theorem nat_cast_im (n : ℕ) : (n : ℂ).im = 0 := by rw [← ofReal_nat_cast, ofReal_im]
 #align complex.nat_cast_im Complex.nat_cast_im
 
 @[simp, norm_cast]
-theorem of_real_int_cast (n : ℤ) : ((n : ℝ) : ℂ) = n :=
+theorem ofReal_int_cast (n : ℤ) : ((n : ℝ) : ℂ) = n :=
   map_intCast ofReal n
-#align complex.of_real_int_cast Complex.of_real_int_cast
+#align complex.of_real_int_cast Complex.ofReal_int_cast
 
 @[simp, norm_cast]
-theorem int_cast_re (n : ℤ) : (n : ℂ).re = n := by rw [← of_real_int_cast, of_real_re]
+theorem int_cast_re (n : ℤ) : (n : ℂ).re = n := by rw [← ofReal_int_cast, ofReal_re]
 #align complex.int_cast_re Complex.int_cast_re
 
 @[simp, norm_cast]
-theorem int_cast_im (n : ℤ) : (n : ℂ).im = 0 := by rw [← of_real_int_cast, of_real_im]
+theorem int_cast_im (n : ℤ) : (n : ℂ).im = 0 := by rw [← ofReal_int_cast, ofReal_im]
 #align complex.int_cast_im Complex.int_cast_im
 
-@[simp, norm_cast]
-theorem of_real_rat_cast (n : ℚ) : ((n : ℝ) : ℂ) = n :=
+@[simp]
```

Comment:
where did the norm_cast go?

Reply:
Accidental deletion (rather lack of restoring)